### PR TITLE
refactor(editor): split off api definitions into seperate libraries

### PR DESCRIPTION
**Pull request by Itrulia**
Allows to re-use it in different libraries in the future such as the membership/editor library

### CODE REVIEW

File: libs/editor/api-v2/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# editor-api-v2
```
Comment (michael-scheurer) [inline libs/editor/api-v2/README.md:1]: what is the intention of api v2?

File: libs/editor/api/src/lib/graphql.ts [hunk 2]
```diff
@@ -1,8 +1,7 @@
 // THIS FILE IS AUTOGENERATED, EDIT WITH CAUTION
+import {Node} from 'slate'
```
[the remark this answers is not in the document]
Comment (Itrulia) [reply libs/editor/api/src/lib/graphql.ts:2]: Yeah it's because it wasn't in the prettier ignore

File: libs/editor/api-v2/README.md (hunk 1 above)
Comment (Itrulia) [reply libs/editor/api-v2/README.md:1]: Comes from https://github.com/wepublish/wepublish/pull/810, it's the version we are working on right now to use nestjs.

**Review by michael-scheurer — DISMISSED** [review]

File: apps/editor/src/app/routes/articleList.tsx [hunk 3]
```diff
@@ -17,7 +10,40 @@ import {
   useDeleteArticleMutation,
   useDuplicateArticleMutation,
   useUnpublishArticleMutation
-} from '../api'
+} from '@wepublish/editor/api'
+import {useEffect, useEffect, useMemo, useMemo, useState, useState} from 'react'
```
Comment (Itrulia) [inline apps/editor/src/app/routes/articleList.tsx:14]:
```suggestion
import {useEffect, useMemo, useMemo, useState, useState} from 'react'
```

**Pull request merged**